index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="referrer" content="no-referrer">
+    <meta http-equiv="Content-Security-Policy" content="%VITE_CSP_META%" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+    <!-- WalletConnect uses the first image found in the head metadata for their wallet connect SDKs (web, kotlin, swift) -->
+    <link rel="apple-touch-icon" sizes="192x192" href="/icon-192x192.png" integrity="%VITE_SRI_ICON_192X192_PNG%" crossorigin="anonymous" />
+    <link rel="apple-touch-icon" sizes="512x512" href="/icon-512x512.png" integrity="%VITE_SRI_ICON_512X512_PNG%" crossorigin="anonymous" />
+    <link rel="icon" type="image/png" href="/favicon.png" integrity="%VITE_SRI_FAVICON_PNG%" crossorigin="anonymous" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" integrity="%VITE_SRI_FAVICON_SVG%" crossorigin="anonymous" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no,viewport-fit=cover" />
+    <meta property="og:title" content="ShapeShift DAO">
+    <meta property="og:image" content="/ss-thumb.jpg">
+    <meta property="og:type" content="website" />
+    <meta property="og:image:width" content="1200"/>
+    <meta property="og:image:height" content="630"/>
+    <meta name="twitter:card" content="summary_large_image">
+    <meta
+      name="description"
+      content="ShapeShift DAO | Your Web3 & DeFi Portal"
+    />
+    <link rel="preconnect" href="https://rsms.me/">
+    <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="/manifest.json" integrity="%VITE_SRI_MANIFEST_JSON%" crossorigin="anonymous" />
+    <link rel="prefetch" href="/env.json" as="fetch" crossorigin="anonymous" />
```

Comment:
not seeing this generated artifact being served anymore. not sure how important it is tbh, but something I noticed.

Reply:
I couldn't find any utility either, decided to remove it! [780fc35](https://github.com/shapeshift/web/pull/8924/commits/780fc358dd698fa1d4f35579acc5f620dc5da98a)